libevmasm/AssemblyItem.cpp

```diff
@@ -208,11 +211,17 @@ string AssemblyItem::toAssemblyText() const
 		text = string("data_") + util::toHex(data());
 		break;
 	case PushSub:
-		text = string("dataOffset(sub_") + to_string(static_cast<size_t>(data())) + ")";
-		break;
 	case PushSubSize:
-		text = string("dataSize(sub_") + to_string(static_cast<size_t>(data())) + ")";
+	{
+		vector<string> subPathComponents;
+		for (size_t subPathComponentId: _assembly.decodeSubPath(static_cast<size_t>(data())))
+			subPathComponents.emplace_back("sub_" + to_string(subPathComponentId));
+		text =
+				(type() == PushSub ? string("dataOffset(") : string("dataSize(")) +
+				solidity::util::joinHumanReadable(subPathComponents, ".") +
+				")";
```

Comment:
Bad indent, plus moving the opening parentheses off the first line makes it easier to parse visually.
```suggestion
		text =
			(type() == PushSub ? string("dataOffset") : string("dataSize")) +
			"(" +
			solidity::util::joinHumanReadable(subPathComponents, ".") +
			")";
```

Reply:
Also, you can use a suffix instead of string constructor: `"dataOffset"s` instead of `string("dataOffset")`.

Reply:
The indent is still wrong. I think you missed my first comment :)